### PR TITLE
Propose re-naming 2 indicators for clarity.

### PR DESCRIPTION
**Pull request by Lucyeoh**
Having two such short-hand indicators that include the word "documentation" is confusing, especially in presentations or situations where the whole standard isn't displayed. Changing "documented ownership" to "Clear Ownership" I think captures the same meaning but while reducing confusion.

2.  "mandatory dependencies" is the only indicator short-hand that doesn't describe what the requirement actually is. I.e. DPGs must have "relevance to sdg's, use of an appropriate license, defined ownership etc." changing this to "independence" or even "independence from closed components" brings it into the same format as the others and I think increases comprehension and simplifies communication of the standard.

### CODE REVIEW

**Comment by Lucyeoh** [comment]
Revised proposal based on discussion with @christer-io @nathanbaleeta : 

3. Documentation of ownership to  "Clear Ownership" 
4. Mandatory dependencies to "Platform Independence"

Waiting on review from @lacabra

**Review by christer-io — APPROVED** [review]

**Comment by nathanbaleeta** [comment]
Took action on the proposed changes and opened a [pull request](https://github.com/DPGAlliance/DPG-Standard/pull/41). Waiting on review from @lacabra.

**Review by nathanbaleeta — COMMENTED** [review]
This PR duplicates #41 which addressed the renaming of both indicators hence closing the PR since it was resolved.

**Pull request closed without merging**
